Libraries/Opc.Ua.Server/NodeManager/MasterNodeManager.cs

```diff
@@ -459,6 +459,90 @@ public void RegisterNamespaceManager(string namespaceUri, INodeManager nodeManag
             }
         }
 
+        /// <summary>
+        /// Unregisters the node manager as the node manager for Nodes in the specified namespace.
+        /// </summary>
+        /// <param name="namespaceUri">The URI of the namespace.</param>
+        /// <param name="nodeManager">The NodeManager which no longer owns nodes in the namespace.</param>
```

Comment:
Is there an API to get the INodeManager for a namespace? Just wonder how an app can get a hold on it, when it didn't create the nodemanager.

Reply:
There currently is not that I am aware of...

Given the current state of the code, I think one would have to:

1.  Access the Node Managers being managed by `MasterNodeManager` via `MasterNodeManager.NodeManagers` property
2. Access the namespace indexes owned by the retrieved `CustomNodeManager2` instances via `CustomNodeManager2.NamespaceIndexes` property in search of the desired namespace

This would be making an assumption that all node managers have been registered as namespace managers correctly.